### ISSUE TITLE
Integrate backend

### ISSUE THREAD

**Issue by waltermvp** [epiphanyapps/ProductivitySuite#1]
We will currently use a JAWS backend to be used for demos. All Routers should be CRUD compliant.

Begin to integrate backend into Productivity Suite Pod:

- [x] Todo
- [ ] Users
